requirements_all.txt

```diff
@@ -235,6 +235,7 @@ aiopvapi==1.6.19
 # homeassistant.components.pvpc_hourly_pricing
 aiopvpc==3.0.0
 
+# homeassistant.components.lidarr
 # homeassistant.components.sonarr
 aiopyarr==22.2.2
```

Comment:
Since `lidarr` and `sonarr` use the same underlying package, wouldn't the `sonarr` integration work as well? Or are the API's different, but implemented in the same package.

Reply:
They are very similar but will run in different instances. I'm currently running this and Radarr as custom alongside Sonarr working just fine.

Reply:
The reasion why I asked was to see if it could perhaps just be a single integration, with a mention of 'Sonarr' via [supported brands](https://github.com/home-assistant/architecture/discussions/734).

Reply:
Radarr is currently separate but broken. I would think that would have to be merged as well. These programs are all differently installable by the user hence separate domains. @frenck What is your take?

Reply:
If the API back end is the same, I would opt-in for a single integration with supported brands for Lidarr/Radarr/Readarr/Sonarr. Perhaps a good opportunity to use this new integration, and sunset + migrate the Radarr eventually. (but that would be up to the core maintainers 👍🏻)

Reply:
Since they are different applications/communities building them, let's keep it different integrations. A switch is also very similar to a light, but we still duplicate the code. Eventually everything grows apart. As long as they can re-use the underlying lib, a lot of code is still shared.